### PR TITLE
Parameterize the registry of docker images used in Prow config

### PR DESCRIPTION
**Pull request by adrcunha**

### CODE REVIEW

**Comment by adrcunha** [comment]
/hold

This doesn't work this way.

File: ci/prow/make_config.go [hunk 1]
```diff
@@ -1309,6 +1313,14 @@ func main() {
 	if len(flag.Args()) != 1 {
 		log.Fatal("Pass the config file as parameter")
 	}
+
+	flakesreporterDockerImage = *dockerImagesBase + "/" + *flakesreporterDockerImageName
```
Comment (chaodaiG) [inline ci/prow/make_config.go]: nit, use path.Join instead of string concatenation since `*dockerImagesBase` could end with `/`

**Comment by chaodaiG** [comment]
Left one nit, will lgtm once merge conflict is resolved

**Comment by chaodaiG** [comment]
/lgtm
/hold
feel free unhold if you decide to ignore my nit

File: ci/prow/make_config.go (hunk 1 above)
Comment (adrcunha) [reply ci/prow/make_config.go]: Done.

**Pull request merged**